test/SparqlAntlrParserTest.cpp

```diff
@@ -2429,40 +2429,6 @@ TEST(SparqlParser, Update) {
                     "PREFIX foo: <foo/> INSERT DATA { foo:a foo:b foo:c }"))));
 }
 
-TEST(SparqlParser, QueryOrUpdate) {
-  auto expectQuery =
-      ExpectCompleteParse<&Parser::queryOrUpdate>{defaultPrefixMap};
-  auto expectQueryFails = ExpectParseFails<&Parser::queryOrUpdate>{};
-  auto Iri = [](std::string_view stringWithBrackets) {
-    return TripleComponent::Iri::fromIriref(stringWithBrackets);
-  };
-  // Empty queries (queries without any query or update operation) are
-  // forbidden.
-  auto emptyMatcher = ::testing::HasSubstr("Empty quer");
-  expectQueryFails("", emptyMatcher);
-  expectQueryFails(" ", emptyMatcher);
-  expectQueryFails("PREFIX ex: <http://example.org>", emptyMatcher);
-  expectQueryFails("### Some comment \n \n #someMoreComments", emptyMatcher);
-  // Hit all paths for coverage.
```

Comment:
With this PR, 
what happens for such empty (or prefix only) queries?
Is this behavior useful/documented/tested somewhere?

Reply:
Theses functionally empty operations can only happen with updates. These are now omitted, because the updates already return a vector. The updated tests are now back in the `Update` test.